### DISCUSSION TITLE
Command parsing with manual nmap upload

### DISCUSSION THREAD

**Discussion by kooki3monst4r** [reconmap/reconmap#73] [Q&A] [answered]
Is command parsing supposed to work with a manually uploaded nmap file? I haven't been able to get rmap to work yet (I think it might be a networking issue on my end) so I uploaded an nmap file instead, but it looks like it wasn't parsed and added as a target.

I double checked that my command had nmap selected as the output and the task was assigned to the project I was working on. I didn't see anything in the documentation about doing this manually, so I thought I would ask.

**Comment by santiagolizardo** [comment, marked as answer]
Hi @kooki3monst4r ; Yes this is supposed to work. Command results can be uploaded automatically by the `rmap` command but you always have the option to upload a result manually. Let me know if you find issues with this I will be happy to troubleshoot them with you. Regards
